controllers/news-controllers.js

```diff
@@ -72,3 +72,18 @@ exports.createComment = (request, response, next) => {
         next(error);
     })
 }
+
+exports.patchArticleVotes = (request, response, next) => {
+    const { article_id } = request.params;
+    const { inc_votes } = request.body;
+
+     if(typeof inc_votes !== 'number'){
+        return next({ status: 400, msg: 'Invalid type'})
```

Comment:
psql can handle it for you when the value is not the right datatype. Also if you want to use customs error handling, better to Promise.reject rather than use an object to invoke next